bcipy/tasks/rsvp/main_frame.py

```diff
@@ -143,6 +148,8 @@ def __init__(self,
         # Items shown in every sequence
         self.seq_constants = seq_constants
 
+        self.criteria_evaluator = criteria_evaluator or CriteriaEvaluator.default()
```

Comment:
I think I originally did it this way to avoid putting the source code of the CriteriaEvaluator the DecisionMaker, since it must be declared before its reference. But I agree that it's better as a default param if you don't mind all this coming first. We could also factor this out into a separate module.